src/server.c

```diff
@@ -1313,6 +1313,9 @@ int serverCron(struct aeEventLoop *eventLoop, long long id, void *clientData) {
      * handler if we don't return here fast enough. */
     if (server.watchdog_period) watchdogScheduleSignal(server.watchdog_period);
 
+    /* Update the time cache, primarily to update daylight info. */
+    updateCachedTime(1);
```

Comment:
Here we just want to update the daylight, maybe extract the code as `updateDaylightWithUs()` and just call it?

Reply:
yes, it is a method, i ever tried to use `updateCachedTimeWithUs(1, server.ustime)` to avoid update server time, but i found the comment of `updateCachedTime` as below, so i don't use this internal function.

```
/* This function should be fast because it is called at every command execution
 * in call(), so it is possible to decide if to update the daylight saving
 * info or not using the 'update_daylight_info' argument. Normally we update
 * such info only when calling this function from serverCron() but not when
 * calling it from call(). */
```